kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/document/datadictionary/PurchaseOrderRetransmitDocument.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?><beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xmlns:dd="http://rice.kuali.org/dd" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd         http://rice.kuali.org/dd         http://rice.kuali.org/dd/dd.xsd">
+
+
+  <bean id="PurchaseOrderRetransmitDocument" parent="PurchaseOrderRetransmitDocument-parentBean">
+    <property name="documentClass" value="edu.arizona.kfs.module.purap.document.PurchaseOrderRetransmitDocument"/>
+    <property name="baseDocumentClass" value="edu.arizona.kfs.module.purap.document.PurchaseOrderRetransmitDocument"/>
+    <property name="documentPresentationControllerClass" value="edu.arizona.kfs.module.purap.document.authorization.PurchaseOrderRetransmitDocumentPresentationController"/>
```

Comment:
Please remove. The edu.arizona version of this presentation controller is not needed.